Northstar.Client/mod/scripts/vscripts/spectator_pov_button_fix.nut

```diff
@@ -0,0 +1,12 @@
+global function SpectatorPovButtonFixInit
+
+void function SpectatorPovButtonFixInit(){
+	RegisterButtonPressedCallback(MOUSE_MIDDLE, SwitchPov)
```

Comment:
This doesnt support controllers?

Reply:
Never used a controller for more than 15 minutes so I don't really know what the input for controller should be